main.js

```diff
@@ -0,0 +1,34 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
```

Comment:
If this is the function that renders, the variable name should be `render` (without the 'er' at the end).